DuckDuckGo/AutocompleteViewController.swift

```diff
@@ -23,7 +23,12 @@ import Core
 class AutocompleteViewController: UIViewController {
     
     struct Constants {
-        static let debounceDelay = 0.3
+        static let debounceDelay: TimeInterval = {
+            if #available(iOS 13.0, *) {
```

Comment:
Do we need to split the logic here? Will 100ms work for lower version too?

Reply:
I'd prefer to split it, as on iOS 12 this is workaround for a crash within Apple SDK. It is supposed to be fixed on iOS 13. If this change does not result in crashes we can consider removing this from iOS 13+ completely.

Reply:
I may need some more context here. The previous workaround was the debounce right? Did the 300ms delay eliminate or reduce the issue?

Reply:
Thanks for the clarification, if we still see these then it makes sense to limit the debounce reduction to iOS 13 only 👍

Reply:
Thanks for the review!